hazelcast-jet-core/src/test/java/com/hazelcast/jet/pipeline/test/TestSourcesTest.java

```diff
@@ -75,9 +76,11 @@ public void test_itemStream_withWindowing() throws Throwable {
          .window(WindowDefinition.tumbling(1000))
          .aggregate(AggregateOperations.counting())
          .apply(assertCollectedEventually(10, windowResults -> {
-             // find any window that has 10 items, some may be incomplete due to hiccups
-             boolean matched = windowResults.stream().anyMatch(r -> r.result() == itemsPerSecond);
-             assertTrue("Did not find any window with 10 items: " + windowResults, matched);
+             Range<Integer> range = Range.closed(itemsPerSecond - 1, itemsPerSecond + 1);
```

Comment:
I'd prefer not to depend on Guava for tests, you can rewrite this easily using `IntStream.range()`

Reply:
Ok